McCorrections/include/JetPuIdWeights.h

```diff
@@ -0,0 +1,40 @@
+/*! Jet PU Id weight.
+This file is part of https://github.com/hh-italian-group/hh-bbtautau. */
+
+#pragma once
+
+#include "h-tautau/Core/include/AnalysisTypes.h"
+#include "WeightProvider.h"
+
+namespace analysis {
+namespace mc_corrections {
+
+struct JetInfo {
```

Comment:
You don't need `struct JetInfo`. The formula is simple and can be calculated directly in `JetPuIdWeights::Get`